packages/react-ui-components/src/Dialog/DialogManager.ts

```diff
@@ -0,0 +1,56 @@
+export interface EventRoot {
+    addEventListener: Document['addEventListener'];
+    removeEventListener: Document['removeEventListener'];
+}
+
+export interface Dialog {
+    close: () => void;
+}
+
+/**
+ * @TODO
+ * Replace this class with a proper dialog management solution
```

Comment:
i think the implementation is _proper_ enough. Just want to guard against todo's that we aint gonna solve (we have already a lot of them and some with higher priority)

Reply:
Can't argue that :D

Makes sense, I removed the comment.